src/Microsoft.IIS.Administration.AccessManagement/Microsoft.IIS.Administration.AccessManagement.csproj

```diff
@@ -12,6 +13,10 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="MicroBuild.Core" Version="0.2.0" />
```

Comment:
Can you define the package version number in the common props file as well? You avoid the risk of them going out of sync